protos/sigstore_bundle.proto

```diff
@@ -48,12 +50,12 @@ message TimestampVerificationData {
 // signatures.
 message VerificationMaterial {
         oneof content {
-                dev.sigstore.common.v1.PublicKeyIdentifier public_key = 1;
-                dev.sigstore.common.v1.X509CertificateChain x509_certificate_chain = 2;
+                dev.sigstore.common.v1.PublicKeyIdentifier public_key = 1 [(google.api.field_behavior) = REQUIRED];
+                dev.sigstore.common.v1.X509CertificateChain x509_certificate_chain = 2 [(google.api.field_behavior) = REQUIRED];
         }
         // This is the inclusion promise and/or proof, where
         // the timestamp is coming from the transparency log.
-        repeated dev.sigstore.rekor.v1.TransparencyLogEntry tlog_entries = 3;
+        repeated dev.sigstore.rekor.v1.TransparencyLogEntry tlog_entries = 3 [(google.api.field_behavior) = REQUIRED];
```

Comment:
The tlog isn't technically required, since you might be running in a private environment where you don't use tlogs.